### PR TITLE
Make crates.bzl public

### PR DESCRIPTION
**Pull request by UebelAndre**
this exports `crates.bzl` so it can be directly added to a [bzl_library](https://github.com/bazelbuild/bazel-skylib/blob/master/bzl_library.bzl) for use in [stardoc](https://github.com/bazelbuild/stardoc). Without exporting this file, stardoc is not able to render documentation for files loading the `crates.bzl` because it can't be added as a dependency because it's not visibile.

```python
load("@bazel_skylib//:bzl_library.bzl", "bzl_library")

bzl_library(
    name = "something",
    srcs = ["//remote/cargo_workspace/cargo:crates.bzl"],
)
```

The above target added to `examples/BUILD` would fail with:
```output
ERROR: /Code/cargo-raze/examples/BUILD:3:12: no such target '//remote/cargo_workspace/cargo:crates.bzl': target 'crates.bzl' not declared in package 'remote/cargo_workspace/cargo'; however, a source file of this name exists.  (Perhaps add 'exports_files(["crates.bzl"])' to remote/cargo_workspace/cargo/BUILD?) defined by /Code/cargo-raze/examples/remote/cargo_workspace/cargo/BUILD.bazel and referenced by '//:something'
ERROR: Analysis of target '//:something' failed; build aborted: Analysis failed
```

### CODE REVIEW

File: impl/src/rendering/bazel.rs [hunk 1]
```diff
@@ -351,16 +389,10 @@ impl BuildRenderer for BazelRenderer {
 
     // Ensure there is always a `BUILD.bazel` file to accompany `crates.bzl`
     let crates_bzl_pkg_file = path_prefix.as_path().join("BUILD.bazel");
```
Comment (acmcarther) [inline impl/src/rendering/bazel.rs:391]: Move this into the render_crates_bzl_package_file function?

File: impl/src/rendering/templates/workspace.BUILD.template [hunk 2]
```diff
@@ -4,6 +4,13 @@ package(default_visibility = ["//visibility:public"])
 licenses([
     "notice",  # See individual crates for specific licenses
 ])
+{%- if is_remote_mode %}
+
+# Export file for Stardoc support
+exports_files([
```
Comment (acmcarther) [inline impl/src/rendering/templates/workspace.BUILD.template:10]: Can this be deduplicated against the inlined contents in the render function? This duplication seems crappy

**Review by acmcarther — CHANGES_REQUESTED** [review]

**Comment by acmcarther** [comment]
Nit: This PR should be titled after what it's doing, not the purpose. Hence, "Make crates.bzl public", not "Add stardoc support".

File: impl/src/rendering/templates/workspace.BUILD.template (hunk 2 above)
Comment (UebelAndre) [reply impl/src/rendering/templates/workspace.BUILD.template]: Deduplicated it

File: impl/src/rendering/bazel.rs (hunk 1 above)
Comment (UebelAndre) [reply impl/src/rendering/bazel.rs]: Done

**Comment by UebelAndre** [comment]
@acmcarther Updated and ready for review.

**Review by acmcarther — APPROVED** [review]

**Pull request merged**